lib/routes/routeBackbeat.js

```diff
@@ -1117,6 +1122,77 @@ function listLifecycle(request, response, userInfo, log, cb) {
     });
 }
 
+const indexesForFeature = {
```

Comment:
It would be a good idea to make our API more modular to accommodate future needs. 
While we may only need to create a specific index at the moment, there may be other types of indexes that we may need to create in the future. By making our API more modular, we can ensure that it will be more flexible and adaptable to changing requirements.

Some potential benefits of making our API more modular include:

**Scalability**: By building a modular API, we can easily add new functionality as our needs evolve, without having to completely rewrite our codebase.

**Reusability**: A modular API can be more easily reused in other applications or contexts, which can save time and effort in future projects.

**Flexibility**: With a more modular API, we can change and update specific modules without affecting the rest of our codebase, which can make it easier to maintain and debug.

Regarding the security concern, please note that only users who are authenticated and authorized will have the ability to create indexes. Currently, we rely on a single permission, s3:ReplicateObject, to authorize all Backbeat API calls. However, this is only a temporary measure and we plan to implement more granular permissions in the future.